src/engine/Settings.cs

```diff
@@ -175,6 +176,14 @@ private Settings()
     /// </summary>
     public SettingValue<bool> CheatsEnabled { get; set; } = new SettingValue<bool>(false);
 
+    /// <summary>
+    ///   The current controls of the game.
+    ///   Key = InputMap name.
+    ///   Value = The events associated with the control.
```

Comment:
Could you explain this a bit more elaborately.
Also this should explicitly say `key events` or input events, as just events is a bit confusing to me as it could be something else than the configured input keys.